src/lib/chat/matrix-client.ts

```diff
@@ -36,6 +36,10 @@ export class MatrixClient implements IChatClient {
   private events: RealtimeChatEvents = null;
   private connectionStatus = ConnectionStatus.Disconnected;
 
+  private static readonly RELATES_TO = 'm.relates_to';
+  private static readonly NEW_CONTENT = 'm.new_content';
+  private static readonly REPLACE = 'm.replace';
```

Comment:
Didn't you recently create a Types file where something like this could be put?

Reply:
Good shout. Extracted.